src/com/owncloud/android/files/FileMenuFilter.java

```diff
@@ -215,8 +215,12 @@ private void filter(List<Integer> toShow, List <Integer> toHide) {
                 (capability.getFilesSharingApiEnabled().isTrue() ||
                         capability.getFilesSharingApiEnabled().isUnknown()
                 );
+
+        boolean notAllowResharing = isSingleSelection() && anyFileSharedWithMe() &&
```

Comment:
@davigonz , is the ```isSingleSelection()``` part really needed in this condition? Would appropriate to allow resharing multiple files if resharing is disabled by admin?